x-pack/plugins/security_solution/server/lib/timeline/saved_object/notes/index.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FrameworkRequest } from '../../../framework';
+import { PageInfoNote, ResponseNote, ResponseNotes, SortNote } from '../../../../graphql/types';
+import { SavedNote, NoteSavedObject } from '../../../../../common/types/timeline/note';
+
+export * from './saved_object';
+export interface Note {
```

Comment:
Wonder if the interface should be changed to Notes to match the file name and the fact this can get multiple? (A la how mongo pluralizes collections)